src/main/java/com/jcabi/xml/ClasspathInput.java

```diff
@@ -121,10 +121,12 @@ public String getEncoding() {
     @Override
     public String getStringData() {
         try {
-            return IOUtils.toString(
+            final String data = IOUtils.toString(
                 this.stream,
                 Charset.forName("UTF-8")
             );
+            this.stream.close();
```

Comment:
@longtimeago I also thought about that, but then again, Yegor declared it a bug (the fact that the stream is left open). What should I do?